.github/scripts/testVersionCompatible.py

```diff
@@ -0,0 +1,380 @@
+          encrypt_algo = st.sampled_from(['aes256gcm-rsa','chacha20-rsa']),
+          trash_days=st.integers(min_value=0, max_value=10000), 
+          hash_prefix=st.booleans(), 
+          force = st.booleans(), 
+          no_update = st.booleans(),
+          meta_url=st.sampled_from(META_URLS),
+          )
+    def format(self, juicefs, block_size, capacity, inodes, compress, shards, storage, encrypt_rsa_key, encrypt_algo, trash_days, hash_prefix, force, no_update, meta_url):
+        print('start format')
+        options = [juicefs, 'format',  meta_url, JuicefsMachine.VOLUME_NAME]
+        if not self.formatted:
+            options.extend(['--block-size', str(block_size)])
+            options.extend(['--compress', compress])
+            options.extend(['--shards', str(shards)])
+            options.extend(['--storage', storage])
+            if hash_prefix:
+                options.append('--hash-prefix')
+        options.extend(['--capacity', str(capacity)])
+        options.extend(['--inodes', str(inodes)])
+        options.extend(['--trash-days', str(trash_days)])
+        
+        if force:
+            options.append('--force')
+        if no_update:
+            options.append('--no-update')
+        if encrypt_rsa_key:
+            if not os.path.exists('my-priv-key.pem'):
+                subprocess.check_call('openssl genrsa -out my-priv-key.pem -aes256  -passout pass:12345678 2048'.split())
+            os.environ['JFS_RSA_PASSPHRASE'] = '12345678'
+            options.extend(['--encrypt-rsa-key', 'my-priv-key.pem'])
+            options.extend(['--encrypt-algo', encrypt_algo])
+        
+        if storage == 'minio':
+            bucket = 'http://127.0.0.1:9000/test-bucket'
+            options.extend(['--bucket', bucket])
+            options.extend(['--access-key', 'minioadmin'])
+            options.extend(['--secret-key', 'minioadmin'])
+        elif storage == 'file':
+            bucket = os.path.expanduser('~/.juicefs/local/')
+            options.extend(['--bucket', bucket])
+        else:
+            assert False
+
+        if not self.formatted:
+            self.clear_storage(storage, bucket)
+            self.flush_meta(meta_url)
+        print(f'format options: {" ".join(options)}' )
+        self.exec_check_call(options)
+        self.meta_url = meta_url
+        self.formatted = True
+        print('format succeed')
+
+    @rule(juicefs=st.sampled_from(JFS_BIN))
+    @precondition(lambda self: self.formatted )
+    def status(self, juicefs):
+        print('start status')
+        output = self.exec_check_output([juicefs, 'status', self.meta_url])
+        uuid = json.loads(output.decode('utf8').replace("'", '"'))['Setting']['UUID']
+        assert len(uuid) != 0
+        if self.mounted:
+            sessions = json.loads(output.decode('utf8').replace("'", '"'))['Sessions']
+            assert len(sessions) != 0 
+        print('status succeed')
+
+    @rule(juicefs=st.sampled_from(JFS_BIN))
```

Comment:
We only provide backwards compatibility, so old version may not mount volume formatted by new version.

Reply:
ok, will check the formatted version when mount.

Reply:
have add backwards check using assume.
<img width="934" alt="image" src="https://user-images.githubusercontent.com/100545994/187879957-d0ec6d46-f8dd-40ab-9e2b-a7c6a527b560.png">